tutor/templates/local/docker-compose.yml

```diff
@@ -45,6 +45,7 @@ services:
       --character-set-server=utf8mb3
       --collation-server=utf8mb3_general_ci
       --binlog-expire-logs-seconds=259200
+      --mysql-native-password=ON
```

Comment:
- Reference for this change?
- This would be needed in k8s as well.

Reply:
The plugins mysql-native-password is disable by default in mysql 8.4.0 that makes us unable to authenticate with user having native password, such as the root user in our case. See [here](https://dev.mysql.com/doc/refman/8.4/en/native-pluggable-authentication.html) for more context.

Reply:
This option seems to be deprecated and during my testing, I found that removing this did not effect anything in my tutor installation. Can you confirm once again if this is required?

Reply:
Yes, I have tested it again. We are able to authenticate with root user without the plugin.